setup/flavors/compose/mailu.env

```diff
@@ -175,3 +175,10 @@ DB_HOST={{ db_url }}
 DB_NAME={{ db_name }}
 {% endif %}
 
+{% if (postgresql == 'external' or db_flavor == 'mysql') and webmail_type == 'roundcube' %}
+ROUNDCUBE_DB_FLAVOR={{ db_flavor }}
```

Comment:
?

```suggestion
ROUNDCUBE_DB_FLAVOR={{ roundcube_db_flavor }}
```

Reply:
I have to clarify here:
The `roundcube_db_flavor` and `db_flavor` are made separated in this PR. However for simplicity, the template generator bind them : `roundcube_db_flavor = db_flavor`. This makes the template generator UI more simple. I considered most of the time people want to have both roundcube and mailu share the same RDBMS. 
Makes sense ?